### PR TITLE
Updated text to remove references to highlighted code that was not highlighted.

### PR DESCRIPTION
**Pull request by psb**
…with the code from the code snippet to make the text consistent.

### CODE REVIEW

**Comment by psb** [comment]
For issue [1211](https://github.com/ocaml/ocaml.org/issues/1211)

File: data/tutorials/wf_02_profiling.md [hunk 1]
```diff
@@ -831,7 +865,7 @@ times each part of the code was called.
 
 Profiling native code is done using your operating system's native
 support for profiling. In the case of Linux, we use `gprof`. An alternative
-is [perf](https://en.wikipedia.org/wiki/Perf_(Linux)), as explained below.
+is [perf](<https://en.wikipedia.org/wiki/Perf_(Linux)>), as explained below.
```
Comment (cuihtlauac) [inline data/tutorials/wf_02_profiling.md]:
```suggestion
is [perf](https://en.wikipedia.org/wiki/Perf_(Linux)), as explained below.
```

Why add angle brackets?

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Comment by tmattio** [comment]
Thanks for the PR @psb! Can you revert the formatting and only include the changes to the text in the PR? Especially the added space between the MDX flags and the code blocks might cause issues when running MDX on the docs.

File: data/tutorials/wf_02_profiling.md (hunk 1 above)
Comment (psb) [reply data/tutorials/wf_02_profiling.md]: Sorry, I didn't notice. I didn't add them intentionally. I think prettier in VSCode reformatted the doc. I will undo the formatting changes.

**Comment by psb** [comment]
I think prettier in VSCode reformatted the doc. I will undo the formatting changes and resubmit.

**Comment by cuihtlauac** [comment]
You need to rebase your branch to avoid changing the `data/*/*.xml` files.

**Comment by psb** [comment]
> You need to rebase your branch to avoid changing the `data/*/*.xml` files.

I think I have completely messed up my git for this branch. Is there an easy fix?

**Comment by cuihtlauac** [comment]
Don't panic :-)

It seems to me your revert commit c1cf6e6850012e356d557bca4ef18742c014361c shouln't have happend. 

You can start by removing merging of  `main`:

```bash
git reset --hard 236733c694224a3ff543d32bc5a10b2f9f88485a
```

And then remove the revert:

```bash
git rebase --onto c1cf6e6850012e356d557bca4ef18742c014361c^ c1cf6e6850012e356d557bca4ef18742c014361c
```

**Comment by psb** [comment]
Followed by `git pull --rebase` and then `git push`?

**Comment by cuihtlauac** [comment]
> Followed by `git pull --rebase` and then `git push`?

I must admit I've never used `git pull --rebase` :-)

Here is what I suggest:

1. Make sure `main` is as fresh as possible:
```bash
git checkout main && git pull origin main
```
2. Move to your branch:
```bash
git checkout missing-code-highlighting
```

3. Remove merging `main`. This brings you back to the commit where you were before merging (all the commits from `main`, at that time, above your forking point are removed) 
```bash
git reset --hard 236733c694224a3ff543d32bc5a10b2f9f88485a
```

4. Remove the revert of https://github.com/ocaml/ocaml.org/pull/1212. 
```bash
git rebase --onto c1cf6e6850012e356d557bca4ef18742c014361c^ c1cf6e6850012e356d557bca4ef18742c014361c
```

5. Make sure you are happy with the changes. 

6. Rebase on `main` (this is optional now)

7. Push to GitHub (assuming your fork of ocaml/ocaml.org is named `fork` on your local machine). Force (`-f`) is required because you've rewritten the history with `reset` and `rebase`. 
```bash
git push -f fork HEAD
```

8. Enjoy a break

**Review by cuihtlauac — APPROVED** [review]
Thank you very much, @psb. This is nice

**Comment by psb** [comment]
Thanks for the help with git, @cuihtlauac. I've learnt many lessons from trying to change two sentences :sweat_smile:

**Pull request merged**